core/src/main/kotlin/com/malinskiy/marathon/analytics/external/graphite/BasicGraphiteClient.kt

```diff
@@ -0,0 +1,70 @@
+package com.malinskiy.marathon.analytics.external.graphite
+
+import com.malinskiy.marathon.execution.AnalyticsConfiguration.GraphiteConfiguration
+import java.io.IOException
+import java.io.PrintWriter
+import java.net.Socket
+import java.net.UnknownHostException
+import java.time.Instant
+
+data class GraphiteMetric(
+    val metricName: String,
+    val metricValue: String,
+    val timestamp: Long = Instant.now().toEpochMilli()
```

Comment:
I copied the code of `BasicGraphiteClient` from [Graphite Kotlin Client](https://github.com/mihkels/graphite-client-kotlin). Unfortunately, the library is not available on Maven Central yet. I'd rather not modify the code, because then it can be easily replaced with the library implementation in the future.

I tried using [graphite-client](https://github.com/savantly-net/graphite-client) before, but for some reason it wasn't sending or fetching any data.

Reply:
gotcha. Can you leave a comment there that this is not original code and leave a link to source? thanks

Reply:
done